### ISSUE TITLE
ERROR Unable to establish one or more of the specified browser connections. This can be caused by network issues or remote device failure.

### ISSUE THREAD

**Issue by McHardex** [DevExpress/testcafe#6556]
### What is your Test Scenario?
I am using testcafe docker image on an apple M1 laptop. I have tried the troubleshooting steps [here](https://testcafe.io/documentation/402838/guides/advanced-guides/use-testcafe-docker-image#troubleshooting) but my tests ARE still not running. I keep getting this error that says:
`ERROR Unable to establish one or more of the specified browser connections. This can be caused by network issues or remote device failure.`
 
### What is your web application and your TestCafe test code?
I have a dockerfile that builds the image as such
```
FROM testcafe/testcafe:latest
USER root
WORKDIR /crawler
COPY . . 
RUN npm i
EXPOSE 1337 1338
```

and I use this command to run the tests
`docker run  -it <image name> 'chromium --headless --no-sandbox' tests/test1.js`

PS: THIS WORKS ON ANOTHER LAPTOP. I AM ONLY EXPERIENCING THIS ISSUE RUNNING ON A M1 APPLE MACBOOK

**Comment by McHardex** [comment]
Thank you @viktoria2506. I'm hoping this will be resolved soon. Looking forward to it.

**Comment by qualityshepherd** [comment]
I'm seeing this on my Azure pipelines with the latest testcafe docker image... I updated to 1.18.1... same problem.

**Comment by qualityshepherd** [comment]
I was able to get my tests running by using an older docker version :1.17.1

**Comment by sabrina-s** [comment]
Had the same error - we managed to fix it by **specifying version 1.17.1 for our testcafe docker image**: https://hub.docker.com/r/testcafe/testcafe/tags (we were previously pulling latest)

**Comment by McHardex** [comment]
@AlexSkorkin There’s an update here. Thank you @sabrina-s
